libs/components/lookup/src/lib/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -121,29 +121,29 @@ export class SkyAutocompleteInputDirective
 
       // Do not mark the field as "dirty"
       // if the field has been initialized with a value.
-      if (this.isFirstChange && this.control) {
-        this.control.markAsPristine();
+      if (this.#isFirstChange && this.#control) {
+        this.#control.markAsPristine();
       }
 
-      if (this.isFirstChange && this._value) {
-        this.isFirstChange = false;
+      if (this.#isFirstChange && this._value) {
+        this.#isFirstChange = false;
       }
     }
   }
 
-  private control: AbstractControl;
+  #control: AbstractControl | undefined;
 
-  private isFirstChange = true;
+  #isFirstChange = true;
 
-  private ngUnsubscribe = new Subject<void>();
+  #ngUnsubscribe = new Subject<void>();
 
-  private _autocompleteAttribute: string;
+  private _autocompleteAttribute: string | undefined;
 
   private _blur = new Subject<void>();
 
   private _disabled = false;
 
-  private _displayWith: string;
+  private _displayWith = '';
```

Comment:
Why not change the properties with underscores to use the private `#` designation as well?

Reply:
I didn't want to override out convention of using the underscore for getter/setter properties.